pennylane_lightning/lightning_qubit/lightning_qubit.toml

```diff
@@ -27,7 +27,7 @@ PauliX                 = { properties = [ "invertible", "controllable", "differe
 PauliY                 = { properties = [ "invertible", "controllable", "differentiable" ] }
 PauliZ                 = { properties = [ "invertible", "controllable", "differentiable" ] }
 PhaseShift             = { properties = [ "invertible", "controllable", "differentiable" ] }
-QubitUnitary           = { properties = [ "invertible",                                  ] }
+QubitUnitary           = { properties = [ "invertible", "controllable", "differentiable" ] }
```

Comment:
What is the meaning of `differentiable` again?

Reply:
Supported in circuit with adjoint differentiation. Apparently that has been possible with QubitUnitary for a few months now?

Reply:
Only if `QubitUnitary` is first decomposed into single-parameter gates (i.e. using ZXZ or ZYZ). And this is only possible for 1,2-qubit gates I think.

Reply:
I mean, if you're using Lightning's adjoint classes.

Reply:
@vincentmr Where does this logic exist in the src code?

Reply:
Any differentiable gate is assumed to be of the form `e**(i*theta*G)` where `theta` is a scalar parameter and `G` is the so-called generator. Eventually one hits this [line](https://github.com/PennyLaneAI/pennylane-lightning/blob/master/pennylane_lightning/core/src/simulators/lightning_qubit/algorithms/AdjointJacobianLQubit.hpp#L309), where the state vector object `mu` tries to apply the generator of the gate. `QubitUnitary` does not have a generator, in Lightning or PennyLane, so the state-vector's `applyGenerator` should complain it cannot apply the generator. It is possible to decompose 1,2-qubit gates using ZYZ [decomposition](https://github.com/PennyLaneAI/pennylane/blob/master/pennylane/ops/op_math/decompositions/single_qubit_unitary.py) however.